src/components/EditorBar/EditorBar.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, ReactNode } from 'react';
+import { Container, ConfirmButton, ChildrenWrapper } from './EditorBar.styles';
+import moment from 'moment';
+import { useTranslation } from 'react-i18next';
+
+import SquareButton from '../shared/buttons/SubmitButton';
+
+type Props = {
+  onFinishEdit: () => void;
+  lastSaveTime: number | null;
+  isSaving: boolean;
+  children?: ReactNode;
+};
+
+const EditorBar: FC<Props> = ({
+  onFinishEdit,
+  lastSaveTime,
+  isSaving,
+  children,
+}) => {
+  const { t } = useTranslation();
+  const timeAgo =
+    lastSaveTime &&
```

Comment:
IMO this is a good candidate for `useMemo` hook - to avoid creation of _moment_ instance for each component re-render.